package.json

```diff
@@ -34,7 +35,7 @@
   },
   "jest": {
     "setupFilesAfterEnv": [
-      "jest-extended/all"
+      "jest-sorted"
```

Comment:
this array needs both `jest-sorted` AND `jest-extended/all`

i think you might have fixed in another branch?
Make sure you keep both when merging